test/e2e/vre-ch31-helpers.js

```diff
@@ -0,0 +1,108 @@
+const mock = require('./mock-helpers');
+
+function initApplicationSubmitMock() {
+  mock(null, {
```

Comment:
not sure how this works- this is creating an mock end point that provides the `value` as the response?

Reply:
That is exactly it. It's confusing to read on first pass.